web/src/testgrid_healthiness-summary.ts

```diff
@@ -0,0 +1,78 @@
+import { LitElement, html, css } from 'lit';
```

Comment:
The name of this file should be all hyphens, I think, to match the others

Reply:
Done